recipe/meta.yaml

```diff
@@ -52,19 +61,30 @@ test:
     - boto3 >=1.2.4
     - packaging
     #- pytest-catchlog
-    - mock  # [py2k]
+    - pytz
+    - pip
   files:
     - test_data
-  commands:
-    - fio --help
-    - fio ls test_data/test.shp
-    - fio info test_data/test.shp
+#  commands:
+#    - pip check
+#    - fio --help
+#    - fio ls test_data/test.shp
+#    - fio info test_data/test.shp
```

Comment:
Could you add `fio --help`, `fio ls test_data/test.shp`, and `fio info test_data/test.shp` to `bld.bat` and `build.sh` too?